met-web/src/components/layout/Footer/index.tsx

```diff
@@ -142,7 +143,7 @@ const Footer = () => {
                     </Stack>
                 </Grid>
                 <Grid item xs={12} sm={6} container justifyContent={'flex-end'} alignItems="flex-end">
-                    <MetBody>© 2023 Government of British Columbia</MetBody>
+                    <MetBody>{translate('footer.copyrightNotice')}</MetBody>
                 </Grid>
             </Grid>
         </Grid>
```

Comment:
There's an `alt` tag in this file that still requires the `translate` function. Any text that could possibly appear to the user on the front-end will need to be translatable. This includes `alt`, `aria-label`, `placeholder`, etc.

Reply:
Updated the alt tag, thanks